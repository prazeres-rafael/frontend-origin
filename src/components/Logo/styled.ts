import styled from 'styled-components';
import LogoBase from '../../assets/svg/logo.svg';
import { palette } from '../../utils/palette';

export const LogoOrigin = styled(LogoBase)({
  margin: 16,
  fill: palette.paragraph.primary,

  '@media (min-width: 768px)': {
    margin: '24px 56px',
  },
});
