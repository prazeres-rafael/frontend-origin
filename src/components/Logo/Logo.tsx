import React from 'react';
import { LogoOrigin } from './styled';

export default function Logo(): React.ReactElement {
  return <LogoOrigin />;
}
