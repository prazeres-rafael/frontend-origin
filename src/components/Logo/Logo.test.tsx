import { render } from '@testing-library/react';
import React from 'react';

import Logo from './Logo';

describe('Logo', () => {
  it('should match the snapshot', () => {
    const { container } = render(<Logo />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
