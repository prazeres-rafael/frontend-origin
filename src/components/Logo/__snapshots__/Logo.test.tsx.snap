// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should match the snapshot 1`] = `
<logo.svg
  class="styled__LogoOrigin-sc-y50cdy-0 kbYqGp"
/>
`;
